Sources/Client/Client/Client+Config.swift

```diff
@@ -41,6 +41,7 @@ extension Client {
                     database: Database? = nil,
                     callbackQueue: DispatchQueue? = nil,
                     logOptions: ClientLogger.Options = []) {
+            assert(!apiKey.isEmpty, "Empty string is not a valid apiKey.")
```

Comment:
Much better than logging

Reply:
But should also be a log IMHO.. Maybe we need the function:
```swift
func assertAndLog(expression: ..., _ message: String) {
  assert(...)
  log...
}
```

Reply:
Very good point. I'll add a function like that to `ClientLogger`.